x-pack/plugins/ingest_manager/server/services/agent_config.ts

```diff
@@ -288,19 +308,18 @@ class AgentConfigService {
         // TEMPORARY as we only support a default output
         ...[
           await outputService.get(soClient, await outputService.getDefaultOutputId(soClient)),
-        ].reduce((outputs, { config: outputConfig, ...output }) => {
-          outputs[output.is_default ? 'default' : output.id] = {
+        ].reduce((outputs, { config: outputConfig, is_default, name, id: outputId, ...output }) => {
```

Comment:
we should probably not spread the whole output here but whitelist what field we need